test_mp_allreduce/get_torch_result.py

```diff
@@ -46,17 +46,44 @@ def _gen_torch_inputs_and_dout(self):
 
     def _cal_torch_res(self, x, dout):
         out = x
+        dout_t = dout
         if self._dtype == "bfloat16":
             out = x.to(dtype=torch.bfloat16)
-            dout = dout.to(dtype=torch.bfloat16)
+            dout_t = dout.to(dtype=torch.bfloat16)
         
         torch_dist.all_reduce(out, group=group)
 
         if self._dtype == "bfloat16":
             out = out.to(dtype=torch.float32)
-            dout = dout.to(dtype=torch.float32)
+            dout_t = dout_t.to(dtype=torch.float32)
         
-        return out, dout
+        return out, dout_t
+
+    def _get_and_compare_torch_result(self):
```

Comment:
why add this?

Reply:
To test whether paddle is as stable as torch